examples/server/index.html.cpp

```diff
@@ -0,0 +1,278 @@
+// Generated file, run deps.sh to update. Do not edit directly
```

Comment:
funny that this leading comment work :laughing:

Reply:
I thought it was minority clever :-)